### PR TITLE
Updated training.md to include note about CLI arguments in the YAML file

### PR DESCRIPTION
**Pull request by unityjeffrey**
### Proposed change(s)

Describe the changes made in this PR.

### Useful links (Github issues, JIRA tickets, ML-Agents forum threads etc.)
MLA-1466 and MLA-1557

### Types of change(s)

- [ ] Bug fix
- [ ] New feature
- [ ] Code refactor
- [ ] Breaking change
- [X] Documentation update
- [ ] Other (please describe)

### Checklist
- [ ] Added tests that prove my fix is effective or that my feature works
- [ ] Updated the [changelog](https://github.com/Unity-Technologies/ml-agents/blob/master/com.unity.ml-agents/CHANGELOG.md) (if applicable)
- [ ] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)
- [ ] Updated the [migration guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Migrating.md) (if applicable)

### CODE REVIEW

File: docs/Training-ML-Agents.md [hunk 1]
```diff
@@ -185,6 +176,57 @@ an old set of configuration files (trainer config, curriculum, and sampler files
...
+CLI arguments (such as `--num-envs`).
+
+Reminder that a detailed description of all the CLI arguments can be found by
+using the help utility:
+
+```sh
+mlagents-learn --help
+```
+
+These additional CLI arguments are grouped into environment, engine and checkpoint
```
Comment (ervteng) [inline docs/Training-ML-Agents.md:191]:
```suggestion
These additional CLI arguments are grouped into environment, engine and checkpoint. The available settings and examples are shown below. 
```

**Comment by unityjeffrey** [comment]
@vincentpierre are you okay if I merge this into the release_10 branch?  made @ervteng requested changes but not in master yet.

**Comment by vincentpierre** [comment]
If you want these changes in release_10, I think it is fine. Do you want to modify the changelog or you don't think it is needed?

**Comment by unityjeffrey** [comment]
> If you want these changes in release_10, I think it is fine. Do you want to modify the changelog or you don't think it is needed?

no, i dont think we need to modify change log.  i will merge.  thx!

**Pull request merged**